src/OrbitGl/TextRenderer.cpp

```diff
@@ -386,7 +390,8 @@ int TextRenderer::GetStringWidthScreenSpace(const char* text, uint32_t font_size
 
   std::size_t len = strlen(text);
   for (std::size_t i = 0; i < len; ++i) {
-    texture_glyph_t* glyph = texture_font_get_glyph(GetFont(font_size), text + i);
+    texture_font_t* font = GetFont(font_size);
+    texture_glyph_t* glyph = MaybeLoadAndGetGlyph(font, text + i);
```

Comment:
Am I getting this right? The only behavioural change happens here? We set `texture_atlas_changed_` to true if needed. And that wasn't the case before, right?

Reply:
Yes, exactly. If a glyph was loaded here, we would not update the texture atlas, and we would not notice later as we would just find it via texture_font_find_glyph (which doesn't trigger an update). So, unless there would be a different texture atlas update triggered (by lucky coincidence), these glyphs would just be missing from our texture.

Reply:
Ok, great. Thanks for the explanation!